src/components/Plugin/Charts/CustomBlock.vue

```diff
@@ -0,0 +1,189 @@
+<script setup>
+import { ref, onMounted } from 'vue';
+import { Chart, registerables } from 'chart.js';
+
+Chart.register(...registerables);
+
+const props = defineProps({
+  space: Object,
+  proposal: Object,
+  votes: Object,
+});
+
+const totalVotesPerDayChart = ref(null);
+const votingPowerPerAddress = ref(null);
+const votingPowerPerDay = ref(null);
+
+const space = props.space;
+const proposal = props.proposal;
+const votes = props.votes;
```

Comment:
This should be put inside a computed property.

You can do it like this: `const votes = computed(() => props.votes.sort((a, b) => a.created - b.created)))

Reply:
Pushed this update.